sno/status.py

```diff
@@ -1,102 +1,206 @@
+import json
+import sys
+
 import click
 import pygit2
 
+from .cli_util import MutexOption
 from .structure import RepositoryStructure
 
 
+EMPTY_REPO_JSON = frozenset({
```

Comment:
the goal being to have an immutable dict-y thing?

I reckon this is too unclear; if you _really_ need an immutable dict you can install https://pypi.org/project/frozendict/

but you're only using this once, and it's uppercased so it's clear it's supposed to be constant. A regular dict seems fine

Reply:
Last release of that thing was in 2016 too, might not be great to use

Reply:
https://pypi.org/project/immutabledict/ is better

Reply:
Found a better compromise. Done